server/main.lua

```diff
@@ -84,7 +102,7 @@ local function resetPlayerName(source)
 end
 
 local function getRadioChannelName(radioChannel)
-    return customRadioNames[tostring(radioChannel)] or Config.RadioChannelsWithName[tostring(radioChannel)] or Config.RadioChannelsWithName[tostring(math.floor(radioChannel))] or radioChannel
+    return customRadioNames[tostring(radioChannel)] or Config.RadioChannelsWithName[tostring(math.floor(radioChannel))] or radioChannel
```

Comment:
Don't change this also as it will lead to another issue of setting a name for frequency like 100.5 not working properly